docs/src/pages/components/data-grid/style/style.md

```diff
@@ -98,6 +98,14 @@ const columns: GridColumns = [
 
 {{"demo": "pages/components/data-grid/style/StylingCellsGrid.js", "bg": "inline"}}
 
+## Using the `classes` object prop
```

Comment:
I don't think that we should document it. This is most often a foot-gun, here is why:

<img width="310" alt="Screenshot 2021-05-14 at 15 00 51" src="https://user-images.githubusercontent.com/3165635/118274265-3246f480-b4c5-11eb-9fc3-6ed14217cc54.png">

Reply:
Well what's the point of doing the feature if we don't want to show it to the users?

Reply:
The main value I think is about having the same API as in the core components. It's only a foot gun because we increase specificity for performance. This mean that developers will lose specificity over the default styles of the grid.

Reply:
When doing the demo it felt like a duplicate. I can remove the demo and just add a link to the page that explains how to use `classes`?